keras/preprocessing/BUILD

```diff
@@ -105,3 +105,28 @@ tf_py_test(
         "//keras/testing_infra:test_combinations",
     ],
 )
+
+tf_py_test(
```

Comment:
Please note that the files where this code lives actually lives in `keras/utils` (e.g. `keras/utils/image_dataset.py`). Please rebase to use those files.